### PR TITLE
1.  Code review small refactor , adding new classes of authorization-api module and other...

### PR DESCRIPTION
**Pull request by AlexKazmirchuk**

### CODE REVIEW

File: softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorithation/impl/router/AuthorizationRouterHandlerImpl.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by sasha on 23.02.17.
+ */
+
+@Component
+public class AuthorizationRouterHandlerImpl extends AbstractRouterHandler implements AuthorizationRouterHandler{
```
Comment (Dominys) [inline softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorithation/impl/router/AuthorizationRouterHandlerImpl.java:21]: @AlexKazmirchuk This must be in AbstractRouterHandler

Comment (AlexKazmirchuk) [reply softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorithation/impl/router/AuthorizationRouterHandlerImpl.java:21]: @Dominys Yes I know , and I did it in latest commits ,you can see it in 4 pull request or h_r_add_classes_br branch

Comment (AlexKazmirchuk) [reply softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorithation/impl/router/AuthorizationRouterHandlerImpl.java:21]: I fixed it in this commit : https://github.com/AlexKazmirchuk/cources.server/commit/0365a39bebfc6c33aff2f1e46601c6be69f97f2d

Comment (Dominys) [reply softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorithation/impl/router/AuthorizationRouterHandlerImpl.java:21]: @AlexKazmirchuk I don't care what is in other PR. Only one PR is responsible for changes made in this PR. If it's made in other PR than it should not be here.

**Pull request closed without merging**